packages/slate-hyperscript/src/index.ts

```diff
@@ -3,7 +3,16 @@ import {
   HyperscriptCreators,
   HyperscriptShorthands,
 } from './hyperscript'
-import { createEditor } from './creators'
+import {
```

Comment:
Exports a bunch of methods from Slate hyperscript that may be used for things like Slate Test Utils hyperscript